app/src/main/java/org/connectbot/ui/screens/console/ConsoleScreen.kt

```diff
@@ -322,6 +329,24 @@ fun ConsoleScreen(
                     bottom = innerPadding.calculateBottomPadding()
                 )
                 .windowInsetsPadding(WindowInsets.imeAnimationTarget)
+                .onPreviewKeyEvent { keyEvent ->
```

Comment:
This seems like the right place to put it; why is there another one in `MainActivity`?

Reply:
I found that I forgot to remove the redundant code. Removed the `MainActivity` one.